examples/vanilla-ts-esm/package.json

```diff
@@ -11,7 +11,7 @@
     "esbuild": "esbuild src/** --bundle --target=es2019 --format=esm --outdir=public/dist",
     "build": "yarn esbuild --minify",
     "predeploy": "yarn run build",
-    "deploy": "gh-pages --dist public --dest ./examples/mux-elements/"
+    "deploy": "gh-pages --dist public --dest ./examples/ix-elements/"
```

Comment:
Nope. Will remove.

Left it here because it _should be_ and I didn't want to forget. In retrospect, we should just remove it and add this back in when we're ready to tackle hosting the examples.